appendices/VK_EXT_custom_border_color.txt

```diff
@@ -0,0 +1,97 @@
+include::{generated}/meta/VK_EXT_custom_border_color.txt[]
+
+*Last Modified Date*::
+    2019-10-14
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+  - Joshua Ashton
+  - Liam Middlebrook, Nvidia
+
+This extension provides cross-vendor functionality to specify a custom border color
+for use when the sampler address mode ename:VK_SAMPLER_ADDRESS_MODE_CLAMP_TO_BORDER is used.
+
+To create a sampler which uses a custom border color
+slink:VkSamplerCreateInfo::pname:borderColor must be set to one of:
+
+  * ename:VK_BORDER_COLOR_CUSTOM_FLOAT_EXT
+  * ename:VK_BORDER_COLOR_CUSTOM_INT_EXT
+  * ename:VK_BORDER_COLOR_CUSTOM_UINT_EXT
+
+When ename:VK_BORDER_COLOR_CUSTOM_FLOAT_EXT, ename:VK_BORDER_COLOR_CUSTOM_INT_EXT
+or ename:VK_BORDER_COLOR_CUSTOM_UINT_EXT is used, applications must provide a
+slink:VkSamplerCustomBorderColorCreateInfoEXT in the pNext chain for
+slink:VkSamplerCreateInfo.
+
+For implementations with a limited number of border colors, they must be reference counted:
+
+  * When a sampler is successfully created
+  * When a sampler is destroyed
+Guranteeing that freeing the last sampler that used a custom border color will also
+free up the slot for another border color to be used.
```

Comment:
Resist the temptation to put spec language in the appendix. Nothing in the extension appendix is considered normative, so this would all need to be duplicated to the spec. The appendix should really just have a summary.

Reply:
Right, I will move this tomorrow. Thanks. :)